hw/dv/tools/vcs/fsm_reset_cov.cfg

```diff
@@ -0,0 +1,10 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is used with -cm_fsmresetfilter
+// Format: signal=<reset_signal_name> case=TRUE/FALSE (indicates reset is high/low active)
+
+signal=rst_n      case=FALSE
```

Comment:
i have a very stupid question. Does this apply to fsm's exclusively..? or is it kind of a blanket application?

Reply:
According to the VCS manual, this is only applied to FSM coverage.